hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -173,6 +173,74 @@ def hpat_pandas_series_shape_impl(self):
     return hpat_pandas_series_shape_impl
 
 
+@overload_method(SeriesType, 'std')
+def hpat_pandas_series_std(self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None):
```

Comment:
```suggestion
def hpat_pandas_series_std(self, axis=None, skipna=True, level=None, ddof=1, numeric_only=None):
```
skipna : bool, default True

Reply:
https://github.com/IntelPython/hpat/issues/233

Reply:
According to the Pandas documentation `skipna=None` by default for `series.std`. So I guided by that.